runtime/vara/src/constants.rs

```diff
@@ -27,7 +27,7 @@ pub mod currency {
     pub const ECONOMIC_CENTIUNITS: Balance = ECONOMIC_UNITS / 100;
 
     /// The existential deposit.
-    pub const EXISTENTIAL_DEPOSIT: Balance = 10 * UNITS; // 10 Vara
+    pub const EXISTENTIAL_DEPOSIT: Balance = UNITS; // 1 Vara
```

Comment:
```suggestion
    pub const EXISTENTIAL_DEPOSIT: Balance = 1 * UNITS; // 1 Vara
```
but clippy will warn

Reply:
yeah, don't like an idea to ignore clippy when its not necessary, moreover there is a comment right in place